README.md

```diff
@@ -82,10 +82,10 @@ vendor
 
 ### Installing the Dependencies
 
-Fulfilling the `arduino_ci` library dependency is as easy as running this command:
-
+Fulfilling the `arduino_ci` library dependency is as easy as running either of these two commands:
```

Comment:
Conceptually, this is `bundle install`.  If you need an alternative that doesn't require admin access, `bundle install --vendor` is the second choice (aligning with #152) followed by `bundle install --path vendor/bundle/or/wherever`.

Reply:
On macOS 10.16, bundle version 1.17.2 gives `Unknown switches '--vendor'` in response to `bundle install --vendor`.  

As may be obvious, I'm not very familiar with Ruby, bundle, etc.

Reply:
Sorry ... my head's in the wrong place. That should be `bundle install --deployment` 😅